test/object_store/test_backup.py

```diff
@@ -107,10 +107,9 @@ async def test_backup_to_non_existent_bucket(manager: ManagerClient, s3_server):
     assert status['state'] == 'failed'
 
 
-@pytest.mark.asyncio
-@skip_mode('release', 'error injections are not supported in release mode')
-async def test_backup_is_abortable(manager: ManagerClient, s3_server):
-    '''check that backing up a snapshot for a keyspace works'''
+async def do_test_backup_abort(manager: ManagerClient, s3_server, 
```

Comment:
Parameters for pytest are usually added by `@pytest.mark.parametrize` fixture. I'm not proposing to make v3 of this PR, but would be good to have such a followup

Reply:
The params to actual tests are @pytest.parameterize. This is a helper routine.